rotaract-appointments.php

```diff
@@ -116,6 +116,7 @@ function init_calendar() {
 				height: "auto",
 				events: %1$s
 			});
+			calendar.setOption("locale", "de");
```

Comment:
Why loading all locales and then only using "de"?

Reply:
Because just copy the minimized JS with all locales has the following advantages:
* easier to enable further locales (no further files needed but just a line of JS)
* Setup / Build process need not be touched
* it is minimized

Reply:
@zimbcode OK with that?